Configuration/config.yml

```diff
@@ -30,3 +30,25 @@ framework:
     assets: ~
     php_errors:
         log: true
+
+# Doctrine Configuration
```

Comment:
Some additional documentation here about whether these vars should be manually set would be useful, particularly their relationship to `parameters.yml` and Apache environment variables.

Reply:
Done and re-pushed.